### DISCUSSION TITLE
translate?

### DISCUSSION THREAD

**Discussion by xrtw** [discord-tickets/bot#114] [Support (Q&A)] [answered]
hello, can i translate to my country language?

**Comment by eartharoid** [comment, marked as answer]
Hi.
The current version of the bot ([v2](https://github.com/eartharoid/DiscordTickets/tree/v2/)) does not have an easy way of translating the bot other than going through all of the code files and searching for text to translate.

I have started work on v3 ([master branch](https://github.com/eartharoid/DiscordTickets/tree/master)) which will have all of the messages in [language files](https://github.com/eartharoid/DiscordTickets/tree/master/src/locales) so it can easily be translated into other languages.

What language do you want to translate the bot into?

**Reply by xrtw** [reply]
Latvia. I see its not so easy, but if i can do that, i will
